source/guide_mosquitto.rst

```diff
@@ -0,0 +1,171 @@
+.. highlight:: console
+
+.. author:: André Birke <https://github.com/abirke>
+.. author:: Tim Hetkämper <https://github.com/transistortim>
+
+.. tag:: iot
+.. tag:: lang-c
+.. tag:: protocol-mqtt
+.. tag:: publish-subscribe
+
+.. sidebar:: About
+
+  .. image:: _static/images/mosquitto.png
+      :align: center
+
+#########
+Mosquitto
+#########
+
+.. tag_list::
+
+Mosquitto_ is an open source MQTT message broker written in C. It also provides the tools ``mosquitto_sub`` and ``mosquitto_pub`` for subscription and publication.
+
+
+License
+=======
+
+All relevant legal information can be found here
+
+  * https://github.com/eclipse/mosquitto/blob/master/LICENSE.txt
+
+
+Prerequisites
+=============
+
+If you want to use Mosquitto with a custom domain you need to set it up first:
+
+.. include:: includes/web-domain-list.rst
+
+
+Installation
+============
+
+Luckily, Mosquitto is preinstalled on Uberspace hosts.
+We just need to configure it (see below).
+
+
+Configuration
+=============
+We're setting up the broker to be reachable via a dedicated TCP port (as opposed to MQTT over WebSockets).
+The communication will be secured using TLS encryption and password authentication.
+
+Step 1
```

Comment:
Please name your steps, e.g. "Copy default config", "Open firewall port", etc. This makes them much easier to find, in case you are only reading part of the guide, are continuing an earlier attempt or are debugging an existing installation. I am aware that older guides use numbered steps, but the has [been updated](https://github.com/Uberspace/lab/commit/9a55c0c79f520ecedd8964fb630e90903c192de) for a couple of months now.

Reply:
Thanks for your feedback. We updated our guide accordingly.